tasks/gulp/compile-scss.js

```diff
@@ -10,19 +10,19 @@ const configPaths = require('../../config/paths.json');
 const destinationPath = require('./destination-path');
 const pkg = require('../../package.json');
 
-const errorHandler = (error) => {
+function errorHandler(error) {
```

Comment:
changed this because when it's a const it's capturing the this to be this package I think - where as we want it to be kind of floating and refer to the context it's run in which will be some EventEmitter type thing that has a this.once https://nodejs.org/docs/latest/api/events.html

the iffy thing is that without this, the whole build blows up straight away when there is any error with css compiling - but without it, it continues on and only fails at the end - might be some tech debt here to generally tidy stuff up